ros2component/ros2component/verb/list.py

```diff
@@ -59,7 +52,7 @@ def main(self, *, args):
                         node=node, remote_container_node_name=args.container_node_name
                     )
                     if any(components):
-                        print(*['{}  {}'.format(c.uid, c.name) for c in components], sep='\n')
+                        print(*['{}    {}'.format(c.uid, c.name) for c in components], sep='\n')
```

Comment:
What motivates the change in indentation?

Reply:
To get closer to a conventionally 4 spaces wide tabulation, but otherwise as arbitrary as the former 2 spaces. 

My intent is to be consistent about indentation while at the same time making it easy to use for someone that's shell scripting. Any thoughts on usability in that regard are welcome

Reply:
Since other verbs use 2 spaces for indentation I would rather keep it with 2 for now.

Reply:
Still pending.

Reply:
Alright, fair enough.